mrmustard/physics/bargmann.py

```diff
@@ -202,3 +202,69 @@ def symplectic2Au(S):
     A = math.block([[A_1, A_2], [A_3, A_4]])
 
     return A
+
+
+def X_of_channel(A):
```

Comment:
Can you type hint `A`?

Reply:
In `representations.py` `A` is usually type hinted as `ComplexMatrix`. Maybe we should use that?